pkg/integration/tests/file/stage_deleted_range_select.go

```diff
@@ -0,0 +1,75 @@
+package file
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var StageDeletedRangeSelect = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Stage a range of deleted files using range select",
+	ExtraCmdArgs: []string{},
+	Skip:         false,
+	SetupConfig: func(config *config.AppConfig) {
+	},
+	SetupRepo: func(shell *Shell) {
+		shell.CreateFileAndAdd("dir1/file-a", "")
```

Comment:
This strikes me as quite verbose. It seems to me that we could just have two files: one deleted, and one not deleted. Thoughts?

Reply:
My intention was just for the test to verify this works in the presence of folders. No preference either way.